### DISCUSSION TITLE
Integrate Hoymiles Inverter into Kostal Smart Energy Meter

### DISCUSSION THREAD

**Discussion by mzurhorst** [tbnobody/OpenDTU#869] [Q&A]
Hi all,

I am operating my HM-600 inverter in parallel to a larger solar roof with Kostal equipement  (Plenticore 10.0,  KSEM).
The KSEM web interface allows the configuration of additional (Kostal) inverters.

Has anybody done such an integration already? -- What would be the right strategy?
(I guess, I will have to fake a Kostal device, sure!)  --
DTU --> MQTT --> HomeAssistant --> <some Modbus magic here> --> KSEM?

Have to admit,  I am today still running AhoyDTU, but I sure can flash to OpenDTU as well  :-)
I came here since I saw a PR regarding Modbus, which raised my interest.

Thanks and best regards,
  Marcus

**Comment by madffm** [comment]
Hi Marcus,

I'm also interested.
Have you already found a solution for this task?

br
Matthias
